Resources/Prototypes/Entities/Structures/Storage/Canisters/gas_canisters.yml

```
@@ -637,6 +637,156 @@
       - !type:DumpCanisterBehavior
   - type: Lock
     locked: true
+    
+- type: entity
```

yamllint (1.35.1)

[warning] wrong indentation: expected 4 but found 0

(indentation)